oshi-core/src/main/java/oshi/hardware/CentralProcessor.java

```diff
@@ -252,11 +252,18 @@
     long[][] getProcessorCpuLoadTicks();
 
     /**
-     * Get the System uptime (time since boot).
+     * Get the Unix time of boot.
      *
-     * @return Number of seconds since boot.
+     * @return Number of seconds since the 1970 epoch.
```

Comment:
I think you switched comments here. :)   The Uptime needs the "number of seconds since last booted" while the BootTime is the Unix time that the system was last booted.

Reply:
whoops! Fixed this